try:
    print(1)
except Exception as result:
    print(result)
else:
    print("if there are no errors, run those code")